lua_tests/mispec_pixbuf_1.lua

```diff
@@ -133,17 +127,17 @@ describe('WS2812 buffers', function(it)
     end)
 
     it:should('mix with strings correctly ', function()
-        buffer1 = ws2812.newBuffer(1, 3) 
-        buffer2 = ws2812.newBuffer(1, 3) 
+        buffer1 = pixbuf.newBuffer(1, pixbuf.TYPE_GRB)
+        buffer2 = pixbuf.newBuffer(1, pixbuf.TYPE_GRB)
 
         buffer1:fill(10,22,54)
         buffer2:fill(10,27,55)
-        buffer1:mix(-257,buffer1:dump(),255,buffer2:dump())
```

Comment:
Did you remove the string capability from pixbuf?

Reply:
Yes, though perhaps for somewhat specious reasoning: it wasn't documented and the test was flirting with OOM.

Reply:
True, Maybe we should document it rather and if necessary split the tests in even more parts. As string buffers are allowed everywhere else it would be inconsistent if it wasn't here.

Reply:
"allowed everywhere else"?